host/3.0/publish-appservice-stage.yml

```diff
@@ -39,11 +45,11 @@ steps:
       set -e
       docker pull $SOURCE_REGISTRY/dotnet:$(PrivateVersion)-appservice
 
-      docker tag $SOURCE_REGISTRY/dotnet:$(PrivateVersion)-appservice $TARGET_REGISTRY/dotnet:$(TargetVersion)-appservice-stage$(StageNumber)
-      docker tag $SOURCE_REGISTRY/dotnet:$(PrivateVersion)-appservice $TARGET_REGISTRY/dotnet:$(TargetVersion)-dotnet3-appservice-stage$(StageNumber)
+      docker tag $SOURCE_REGISTRY/dotnet:$(PrivateVersion)-appservice $TARGET_REGISTRY/dotnet:$(TargetVersion)-$(TAG_SUFFIX)
+      docker tag $SOURCE_REGISTRY/dotnet:$(PrivateVersion)-appservice $TARGET_REGISTRY/dotnet:$(TargetVersion)-dotnet3-$(TAG_SUFFIX)
 
-      docker push $TARGET_REGISTRY/dotnet:$(TargetVersion)-appservice-stage$(StageNumber)
-      docker push $TARGET_REGISTRY/dotnet:$(TargetVersion)-dotnet3-appservice-stage$(StageNumber)
+      docker push $TARGET_REGISTRY/dotnet:$(TargetVersion)-$(TAG_SUFFIX)
+      docker push $TARGET_REGISTRY/dotnet:$(TargetVersion)-dotnet3-$(TAG_SUFFIX)
```

Comment:
@CooperLink - will the new test you added continue to work with the changes in this PR?

Reply:
Thank you for the callout. This will break my script for verification. I will try to find a way to fix it soon. I think I should be able to.